### ISSUE TITLE
Add support for GRPC Protocol in Target Groups

### ISSUE THREAD

**Issue by jeastham1993** [aws/copilot-cli#2812]
Hey there, thanks for putting together such a great CLI. Really helps to abstract away a lot of the complexity around VPC and network 'best practices'.

For a lot of internal services I deploy we use GRPC instead of HTTP, as far as I understand the cloud formation changes required I think it would be relatively straightforward to add support for conditional GRPC for load-balanced and backend services. I'm by no means an expert in the networking side of AWS, so please correct me if wrong. 

Something like

```yml

  TargetGroup:
    Metadata:
      'aws:copilot:description': 'A target group to connect the load balancer to your service'
    Type: AWS::ElasticLoadBalancingV2::TargetGroup
    Properties:
      HealthCheckPath: {{.HTTPHealthCheck.HealthCheckPath}} # Default is '/'.
{{- if .HTTPHealthCheck.SuccessCodes}}
      Matcher: 
        HttpCode: {{.HTTPHealthCheck.SuccessCodes}}
{{- end}}
{{- if .HTTPHealthCheck.HealthyThreshold}}
      HealthyThresholdCount: {{.HTTPHealthCheck.HealthyThreshold}}
{{- end}}
{{- if .HTTPHealthCheck.UnhealthyThreshold}}
      UnhealthyThresholdCount: {{.HTTPHealthCheck.UnhealthyThreshold}}
{{- end}}
{{- if .HTTPHealthCheck.Interval}}
      HealthCheckIntervalSeconds: {{.HTTPHealthCheck.Interval}}
{{- end}}
{{- if .HTTPHealthCheck.Timeout}}
      HealthCheckTimeoutSeconds: {{.HTTPHealthCheck.Timeout}}
{{- end}}
      Port: !Ref ContainerPort
      Protocol: HTTP
{{- if .HTTP.EnableGrpc}}
      ProtocolVersion: GRPC
{{- end}}

```

Again, networking is not my fortay so there may be more to think about. It would be a really helpful addition to an already great tool though.

**Comment by seiichi1101** [comment]
I created PR, which enables to set protocol version.

**Comment by efekarakus** [comment]
This is now released 🎉 in v1.13.0 https://github.com/aws/copilot-cli/releases/tag/v1.13.0 !
